code/iomethods.py

```diff
@@ -267,67 +280,77 @@ def read_data(varName, latName, lonName, userVariables, filelist=None):
     time2store = None
     tempMaskedValueNp = []
 
-    nfiles = len(userVariables.filelist)
+    numberOfFiles = len(userVariables.filelist)
 
     # Crash nicely if there are no netCDF files
-    if nfiles == 0:
+    if numberOfFiles == 0:
```

Comment:
Hey @caocampb, if numberOfFiles is only used here, consider making this a one-liner i.e. if len(userVariables.filelist) == 0: